### PR TITLE
Add API

### PR DESCRIPTION
**Pull request by zieeco**
**In this milestone, I did the following**

- Added the `api.js file` 
- Updated the `index.js file` using the API

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,28 +1,22 @@
...
-
-render(arr);
+form.addEventListener('submit', async () => {
+  const { userName, userScore } = form.elements;
+  const game = {
+    user: userName.value,
+    score: userScore.value,
+  };
+  form.reset();
+  await setUserScore(game);
```
Comment (mikethreels) [inline src/index.js]: - [ ] when trying to add a new score it doesn't actually add the new score, please try to fix this.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Isaac, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (zieeco) [reply src/index.js]: For some reason, the project works well on firefox but don't know why I don't on my chrome browser

Comment (GzimAsani) [reply src/index.js]: Still my score does not appear when I add new score. Also for me it does not work on Firefox browser either. Your fetching data looks fine but I think the problem is when you send the new data which is not being saved on the server. Try to fix this issue

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @zieeco                                      
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [ ] Check the comments below :arrow_double_down: 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this 
You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (zieeco) [reply src/index.js]: Oh! After spending the whole of today debugging this, I just realized i forgot to add the `e.preventDefault()` to the form on submit... 

:+1: 

Thank you all for your review..

**Review by Lameck1 — APPROVED** [review]
Hi, @zieeco,

Well done on addressing the previous review concerns 👏🏿.
The tasks for this milestone are now successfully implemented 💯.
All the best with the next milestone.

You are free to merge this branch :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**